webclient/src/containers/Login/Login.tsx

```diff
@@ -68,6 +69,8 @@ const Login = ({ state, description }: LoginProps) => {
   });
   const [userToResetPassword, setUserToResetPassword] = useState(null);
 
+  const passwordResetToast = useToast({key: 'password-reset-success', children: 'Password Reset Successfully'})
```

Comment:
I love how easy this is to use.

Reply:
Thanks!  I'm pretty happy with it too.  I've grown really enamored with this stateful-provider + hook pattern over the past few years.  The DX has been great.